recipes/krb5/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd src
+
+autoreconf -i
+
+./configure --prefix=$PREFIX
+
+make
+if [ "$PY_VER" == "2.7" ]; then
```

Comment:
There are some tests scripts that won't work on `Python 3` and I am not in the mood to patch them yet. Instead of just building the library on `Python 2.7` I decided to skip the tests on `Python !=2.7` to allow Windows to build properly.